src/api/apiClient.ts

```diff
@@ -0,0 +1,8 @@
+import createClient from 'openapi-fetch'
+import type { paths } from '@/api/schema'
+
+const baseUrl = import.meta.env.VITE_API_BASE_URL ?? ''
```

Comment:
```suggestion
const baseUrl = import.meta.env.VITE_API_BASE_URL
```
`createClient` の `baseUrl` 引数は `string | undefined` を受け付けているので、`?? ''` にとくに意図がなければ消してしまうとよいかも知れません。`VITE_API_BASE_URL` 環境変数が未設定の場合にはどちらでも `http://localhost:5173/api` 以下へのリクエストが飛ぶことが確認できました

Reply:
undefined許容されるならこっちで良さそうですね
修正しました